client/src/components/SignInForm.js

```diff
@@ -21,37 +21,41 @@ const SignInForm = () => {
   };
 
   if (shouldRedirect) {
-    return <Redirect push to="/home" />;
+    return <Redirect push to="/warm-fuzzies" />;
   }
 
   return (
     <form onSubmit={handleSubmit(onSubmit)} className="form">
-      <input
-        className="input-field"
-        type="text"
-        name="email"
-        id="email"
-        placeholder="Email"
-        ref={register({
-          required: "Required",
-          pattern: {
-            value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
-            message: "invalid email address"
-          }
-        })}
-      />
-      {errors.email && errors.email.message}
-      <input
-        className="input-field"
-        type="password"
-        name="password"
-        id="password"
-        placeholder="Password"
-        ref={register({
-          required: "Password Required",
-        })}
-      />
-      <input className="button button-center" type="submit" value="Login" />
+      <div className="">
+        <input
+          className="p-2 rounded-md mb-2 input-field"
+          type="text"
+          name="email"
+          id="email"
+          placeholder="Email"
+          ref={register({
+            required: "Required",
+            pattern: {
+              value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
```

Comment:
this regex should be extracted to a configuration variable - helps to remove the "hard coding" should we need to make use of this regex elsewhere, and having a centralized place for items like this can be helpful for organization purposes.

consider placing it in `config.js`